gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.9-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-bin.zip
```

Comment:
This should (now) be `5.0-rc-1`, which is apparently the first release to officially support JDK 11 (even though everything builds just fine for me under `4.9`).

See https://docs.gradle.org/5.0-rc-1/release-notes.html#java-11-runtime-support